config/config.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Looks like this file may be duplicating the sequelize.js file? Does it make sense to just add the test and production environment to that file? Additionally, I think we would not want the actual credentials showing we would want it to be process.env.DB_username....etc